src/idc/idc.c

```diff
@@ -17,6 +17,7 @@
 #include <sof/lib/cpu.h>
```

Comment:
Care to provide some context and some info about the design of ASM in the commit message. Also, what is ACE?

Reply:
Added a bit more info in commit message. ACE means intel platforms form mtl onwards. Differentiating with previously used CAVS.